modules/get-started/pages/release-notes/helm-charts.adoc

```diff
@@ -12,11 +12,79 @@ See also:
 * xref:upgrade:k-compatibility.adoc[]
 * xref:upgrade:k-rolling-upgrade.adoc[]
 
-== Redpanda chart v5.9.21
+== Redpanda chart v25.1-k8s-beta1
+
+link:https://github.com/redpanda-data/redpanda-operator/blob/release/v25.1.x/charts/redpanda/CHANGELOG.md[Changelog].
+
+See also: xref:deploy:deployment-option/self-hosted/kubernetes/k-25.1-beta.adoc[]
+
+=== Redpanda Console v3
+
+This beta version deploys Redpanda Console v3, which includes unified authentication and authorization between the Console and Redpanda, including user impersonation. For more information, see xref:console:config/security/authentication.adoc[].
+
+This version does not support Redpanda Console v2.
+
+=== Schema validation
+
+The Redpanda Helm chart now includes built-in schema validation to ensure configurations conform to expected standards. With schema validation enabled, you must ensure your overrides include valid fields.
+
+=== Deprecated fields
```

Comment:
```suggestion
=== Removed fields
```

A deprecation would imply that the fields are still available but are staged to be removed. We've straight up removed them. Though they will need to be merged :[

Should we instead just mentioned that fields have been removed and delegate to the change log so we don't have to fret about the merge ordering of the removal PRs and this docs update? All this information _should_ be mentioned in the change log and if not I'm happy to update the changelog as need be.